minutes/2021-08-31.md

```diff
@@ -0,0 +1,69 @@
+
+Date: 2021-08-31
+
+Present:
+
+- Ivar Grimstad (Eclipse Foundation)
+- Jan Westerkamp (iJUG)
+- Emily Jiang (IBM)
+- Kevin Sutter (IBM)
+- Jesse McConnell (Webtide)
+- Scott Marlow(Red Hat)
+- Dmitry Kornilov (Oracle)
+- Thomas Watson (IBM)
+- Scott Stark (Red Hat)
+- Werner Keil (Individual)
+- Jean-Louis Monteiro (Tomitribe)
+- John Clingan (Red Hat)
+- Ed Bratt (Oracle)
+- Lukas Jungmann (Oracle)
+- Otavio Santana
+- Cesar Hernández (Tomitribe)
+
+## Agenda and Minutes
+
+### Jakarta EE 10 (standing agenda item)
+
+* Core Profile PR: https://github.com/jakartaee/specifications/pull/419 
+* Scott will add the release plan for Core to the project web page (gh-pages)
+* Scott will send out email with the target dates from the plan to project leads list
+* No feedback yet to the resolution regarding optional components of a Specification
+  * No new optional features will be allowed
+* Start tracking progress of the specifications
+  * Use the spreadsheet in the spec committee
+  * We will also use a GH project board for this
+  * Will the waves be similar? 
+    * Dependencies between the APIs are the same, so that is expected for EE 10 as well
+    * How far we get with the TCK dependencies (circular) remains to be seen
+* First TCK calls on September 8   
+  * Two calls to cover all time zones
+  * [Calendar link](https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc@group.calendar.google.com) 
+  * [Agenda](https://docs.google.com/document/d/1V1dDLJkd14EDRMPeuI0VzPtU4Lbli8FFBd1pLDLlOrY/edit#)
+* Identify implementations used for ratification in the ballots
+  * TODO: Check if a project board can be used for this mapping
+  * Or add a column to the tracking spreadsheet
+* TCK Guides will need to be updated
+
+### Semantic versioning of component specifications (continued)
+
+* Discussion continued on mailing list
+* No objections regarding semantic versioning
+* Some concerns regarding the dependencies suggested for tooling support (e.g osgi Version annotation)
+* Could be put forward as a recommendation, rather than a requirement
+  * Get feedback from the projects adopting it
+  * Set up some guidelines and ask for projects to volunteer (Config?)
+  * Option to add this under the Jakarta namespace (instead of OSGI) at Jakarta Annotations
+  * Potentially add as a platform requirement at a later stage
+* **Signature tests must still work**
+  * Tooling changes likely to be updated to accept this type of annotations in the API classes
+  * Must have agreement that this is a kind of change we want to do
+* Not in the scope for EE 10
+
+### JavaDocs -- and other ancillary material (i.e. Spec., TCK User Guide - ascii-doc builds) working with JDK 11 (Ed)
+
+* When we run into issues running with JDK11, we will need to start working with the project teams case-by-case
+* Be proactive and run the javadoc+ jobs under JDK11 as soon as possible
+  * Scott will create an issue for the platform javadoc
```

Comment:
If you wish, you could add another bullet after this one...

- Kevin will experiment with Java 11 and get the build poms and Jenkins builds updated appropriately

Reply:
Thanks! Done